VK-Clone/VK-Clone/Services/VKAPIService.swift

```diff
@@ -4,8 +4,33 @@
 import Alamofire
 import RealmSwift
 import SwiftyJSON
+import UIKit
 
 final class VKAPIService {
+    // MARK: - Private Properties
+
+    private let requestAPIPath = "https://api.vk.com/method/"
+
+    // MARK: - Static Methods
+
+    static func getUIImage(_ url: String, completion: @escaping (UIImage?) -> ()) {
+        DispatchQueue.global(qos: .userInteractive).async {
```

Comment:
```suggestion
        DispatchQueue.global.async {
```
(qos: .userInteractive) - зачем если он и так высокий